pkg/controller/migmigration/backup.go

```diff
@@ -117,6 +117,27 @@ func (t *Task) getStageBackup() (*velero.Backup, error) {
 	return t.getBackup(labels)
 }
 
+func (t *Task) getPodVolumeBackupsForBackup(backup *velero.Backup) *velero.PodVolumeBackupList {
+	backupAssociationLabel := map[string]string{
+		velero.BackupNameLabel: backup.Name,
+	}
+	client, err := t.getSourceClient()
+	if err != nil {
+		log.Trace(err)
+		return nil
+	}
+	list := velero.PodVolumeBackupList{}
+	err = client.List(
+		context.TODO(),
+		k8sclient.MatchingLabels(backupAssociationLabel),
+		&list)
+	if err != nil {
+		log.Trace(err)
+		return nil
```

Comment:
why not return an empty list instead?

This can panic if the user of this function does not check for nil, `return &list` is more graceful IMHO

Reply:
+1, for the empty list reference, instead of nil

Reply:
+1, for returning an empty list reference